dc_utils/tests/test_dc_forms.py

```diff
@@ -35,6 +35,6 @@ class Meta:
         mocker.patch("dc_utils.templatetags.dc_forms.render", mock_render)
 
         assert dc_form(element=form) == "rendered_form"
-        mock_render.called_once_with(
+        mock_render.assert_called_once_with(
```

Comment:
This never did anything because `called_once_with` isn't a function. Same with the other similar line in this file.

This was not a visible error when running the tests under python 3.10, but python 3.12 introduces a check for this. Related reading:

- https://github.com/python/cpython/issues/100690
- https://discuss.python.org/t/include-prefix-called-in-list-of-forbidden-method-prefixes-for-mock-objects-in-unsafe-mode/22249